main/libssh/APKBUILD

```diff
@@ -1,32 +1,40 @@
 # Contributor: Carlo Landmeter <clandmeter@gmail.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=libssh
-pkgver=0.7.6
-pkgrel=1
+pkgver=0.8.7
+pkgrel=0
 pkgdesc="Library for accessing ssh client services through C libraries"
-url="http://www.libssh.org/"
+url="https://www.libssh.org"
 arch="all"
-license="LGPL"
-makedepends="zlib-dev openssl-dev cmake doxygen"
+license="LGPL-2.1-or-later BSD-2-Clause"
+makedepends="cmake doxygen openssl-dev zlib-dev"
+checkdepends="cmocka-dev"
 subpackages="$pkgname-dev"
-options="!check"
-source="https://www.libssh.org/files/0.7/libssh-$pkgver.tar.xz
-	fix-includes.patch"
+source="https://www.libssh.org/files/${pkgver:0:-2}/libssh-$pkgver.tar.xz"
```

Comment:
This is a bashism we don't want.

Reply:
Thanks for the suggestion. I had already made the change locally but I was waiting for upstream to provide a fix for the failing test. They have now fix the issue and I have update the PR so it should be ready to go.